stubs/psutil/psutil/__init__.pyi

```diff
@@ -160,13 +160,13 @@ class Process:
     def kill(self) -> None: ...
     def wait(self, timeout: int | None = ...) -> int: ...
 
+_Popen = TypeVar("_Popen", bound="Popen")
```

Comment:
For future reference: Use `_typeshed.Self` in cases like this. Using a bound type for self references is unnecessary.

Reply:
Thanks! 👍 

I will change this later.